schemas/org.w3/PerformanceNavigationTiming/jsonschema/1-0-0

```diff
@@ -0,0 +1,231 @@
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's unload event handler completes in milliseconds."
+        },
+        "domInteractive": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the user agent sets the document's readyState to 'interactive' in milliseconds."
+        },
+        "domContentLoadedEventStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the current document's DOMContentLoaded event handler starts in milliseconds."
+        },
+        "domContentLoadedEventEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's DOMContentLoaded event handler completes in milliseconds."
+        },
+        "domComplete": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the user agent sets the document's readyState to 'complete' in milliseconds."
+        },
+        "loadEventStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's load event handler starts in milliseconds."
+        },
+        "loadEventEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's load event handler completes in milliseconds."
+        },
+        "startTime": {
+            "type": [ "integer", "null" ],
+            "maximum": 0,
+            "minimum": 0,
+            "description": "Returns a DOMHighResTimeStamp with a value of 0."
+        },
+        "entryType": {
+            "type": [ "string", "null" ],
+            "maxLength": 128,
+            "description": "The string 'navigation'."
+        },
+        "redirectCount": {
+            "type": [ "integer", "null" ],
+            "minimum": 0,
+            "maximum": 64,
+            "description": "A number representing the number of redirects since the last non-redirect navigation in the current browsing context."
+        },
+        "type": {
+            "type": [ "string", "null" ],
+            "maxLength": 32,
+            "description": "A string representing the navigation type. Either 'navigate', 'reload', 'back_forward' or 'prerender'."
```

Comment:
I wish our version of JSON Schema didn't complain about using [`examples`](https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-validation-00#section-10.4). :sob:

Reply:
Yeap, it would be great! Thanks for the reference!